app/views/helpers/errorSummary.scala.html

```diff
@@ -35,12 +37,13 @@ <h2 class="h2-heading error-summary-heading" id="errorMessageHeadingText">@Messa
                     }
                 ) { anchor =>
                 <li data-journey='@Messages("gmp.bulk.problem.error.field"):@Messages(formHeaderKey):@error.message'>
-                    <a href="#@{if(focusId.isDefined) focusId.get else anchor}" data-focuses="@{if(focusId.isDefined) focusId.get else error.key}">
+                    <a class="govuk-link" href="#@{if(focusId.isDefined) focusId.get else anchor}" data-focuses="@{if(focusId.isDefined) focusId.get else error.key}">
                         @Messages(error.message)
                     </a>
                 </li>
                 }
             }
-        </ul>
+            </ul>
+        </div>
     </div>
 }
```

Comment:
will this page still be needed after all the pages have been updated? I thought that the new play UI integrated this into it so that a separate page was not needed. This question would apply to any of the pages that use the gmp component vs govuk component.
If this is incorrect, it is okay to close/resolve conversation.